README.md

```diff
@@ -3,17 +3,17 @@ go-structmapper
 
 [![license](https://img.shields.io/github/license/mashape/apistatus.svg?maxAge=2592000)](https://github.com/anexia-it/go-structmapper/blob/master/LICENSE)
 [![GoDoc](https://godoc.org/gopkg.in/anexia-it/go-structmapper.v1?status.svg)](https://godoc.org/gopkg.in/anexia-it/go-structmapper.v1)
-[![Build Status](https://travis-ci.org/anexia-it/go-structmapper.svg?branch=v1)](https://travis-ci.org/anexia-it/go-structmapper)
```

Comment:
GitHub actions also give you a status badge, maybe change to that instead of removing it?